ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -30,6 +31,15 @@ public async Task<IList<Notification>> GetNotificationsAsync()
             return await _context.Notification.ToListAsync();
         }
 
+        public async Task<IList<Notification>> GetNotificationsWithPatientsAsync()
+        {
```

Comment:
So I thought as we are lazy loading the related entities on patients we wouldn't need to explicitly include them.. but when I try to access e.g. notification.PatientDetails.Ethnicity, it's always null unless I include it here. Maybe it's related to the fact that patientdetails is now an owned entity, or I'm just misunderstanding lazy loading/doing something wrong.

In any case, this is fine for now - it's only used on patient index for now, as for the edit pages we just need to fetch the ids, not the whole entity.

Reply:
Cool, that's useful to know - could you add a comment to that effect in the code, too? :)